src/main/scala/io/circe/yaml/snake/CirceJsonConverter.scala

```diff
@@ -0,0 +1,27 @@
+package io.circe.yaml.snake
+
+import io.circe.Json
+
+/**
+  * Converts objects constructed by [[org.yaml.snakeyaml.constructor.BaseConstructor]]
+  * into [[Json]].
+  *
+  * @note You should use [[CirceJsonConverter.default]] if you are using [[CirceJsonConstructor]].
+  */
+trait CirceJsonConverter {
+
+  /**
+    * Convert the constructed value to [[Json]].
+    */
+  def convert(obj: Any): Json
```

Comment:
Must these take `Any`? To me, that type signature suggests this converts anything into `Json`, which is not the documented goal of this abstraction.